app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,11 +26,15 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
```

Comment:
I would pass `article` instead of `a`, it keeps it readable and makes it easier to come back to next year